LayoutTests/fast/snapshot/positioned-static.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html>
+<head>
+<style>
+#box {
+    width: 100px;
+    height: 100px;
+    background-color: blue;
+    color: black;
+}
+</style>
+</head>
+<body>
+    <div id="box">Is this included?</div>
```

Comment:
Please include the testa case with a static position div having an abspos child.